mustang_codegen/lib/src/utils.dart

```diff
@@ -12,6 +12,7 @@ class Utils {
   static const String configFile = 'mustang.yaml';
 
   // Keys in mustang-cli.yaml
+  static const String mustangConfigKey = 'mustangConfig';
```

Comment:
Everything in mustang.yaml is config. We can instead call this key `mustantStateConfig` ?